src/XMakeTasks/Copy.cs

```diff
@@ -298,45 +307,17 @@ private void LogDiagnostic(string message, params object[] messageArgs)
                 destinationFileExists = destinationFileState.FileExists;
             }
 
-            bool hardLinkCreated = false;
+            bool linkCreated = false;
 
-            // If we want to create hard links, then try that first
+            // If we want to create hard or symbolic links, then try that first
             if (UseHardlinksIfPossible)
-            {
-                // Do not log a fake command line as well, as it's superfluous, and also potentially expensive
-                Log.LogMessageFromResources(MessageImportance.Normal, "Copy.HardLinkComment", sourceFileState.Name, destinationFileState.Name);
+                TryCopyViaLink("Copy.HardLinkComment", MessageImportance.Low, sourceFileState, destinationFileState, ref destinationFileExists, ref linkCreated, () => NativeMethods.CreateHardLink(destinationFileState.Name, sourceFileState.Name, IntPtr.Zero /* reserved, must be NULL */));
+            else if (UseSymboliclinksIfPossible)
+                TryCopyViaLink("Copy.SymLinkComment", MessageImportance.High, sourceFileState, destinationFileState, ref destinationFileExists, ref linkCreated, () => NativeMethods.CreateSymbolicLink(destinationFileState.Name, sourceFileState.Name, SymbolicLink.File));
```

Comment:
OK, High verbosity looks much better for that error. Let's make it High for both hard and sym links then :)